### PR TITLE
Add a generator to export entities to JDL

### PR DESCRIPTION
**Pull request by cbornet**

### CODE REVIEW

**Comment by cbornet** [comment]
Writing the tests now

File: generators/export-jdl/index.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        insight: function () {
+            var insight = this.insight();
+            insight.trackWithEvent('generator', 'export-jdl');
+        },
+
+        parseJson: function () {
+            this.log('Parsing entities from .jhipster dir...');
+            try {
+                let entities = {};
+                this.getExistingEntities().forEach( entity => entities[entity.name] = entity.definition );
```
Comment (deepu105) [inline generators/export-jdl/index.js]: I think you can use the map method for this

File: generators/export-jdl/index.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+            } catch (e) {
+                this.log(e.message || e);
+                this.error(`\nError while parsing entities to JDL\n`);
+            }
+        }
+    },
+
+    writing: function () {
+        let content = '// JDL definition for application \'' + this.baseName + '\' generated with command \'yo jhipster:export-jdl\'\n\n' + this.jdl.toString();
+        this.fs.write(this.jdlFile, content);
```
Comment (deepu105) [inline generators/export-jdl/index.js]: you could add a log message to let the user know that the process finished successfully?

File: generators/export-jdl/index.js (hunk 1 above)
Comment (cbornet) [inline generators/export-jdl/index.js]: I tried but map returns an array, not an Object

File: generators/export-jdl/index.js (hunk 2 above)
Comment (cbornet) [inline generators/export-jdl/index.js]: done

**Pull request merged**